client/src/components/Donation/donate-form.tsx

```diff
@@ -332,14 +332,25 @@ class DonateForm extends Component<DonateFormProps, DonateFormComponentState> {
     )}:`;
     const showMinimalPayments = isSignedIn && (isMinimalForm || !isDonating);
 
+    const isAsVariant = !isAVariant;
+
```

Comment:
Do we need this?  It seems simpler to just use `isAVariant`.

Reply:
Removed.